farsbot.py

```diff
@@ -62,10 +76,31 @@ async def on_voice_state_update(self, member, before, after):
                 if member.bot:
                     pass
                 else:
+                    soundPath = ""
+                    match member.id:
```

Comment:
match är en ny feature I python 3.10 som inte släppts ännu så jag hade ändrat till if-else osv alternativt en dict.
`uidmap = {801923008532578354: "vislov_moven.wav"}`
och sen I actionen:
`if member.id in uidmap: get_sound_with_name(uidmap[member.id])`

eller dylikt

Reply:
ab402ab6d4b6d5c15f988df28a31f1fbb5b725ad